### DISCUSSION TITLE
GreptimeDB 社群观察报告 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#108] [Comments]
# GreptimeDB 社群观察报告 | 夜天之书

GreptimeDB 是格睿科技（Greptime）公司研发的一款开源时序数据库，其源代码在 GitHub 平台公开发布。  我从 2022 年开始知道有 GreptimeDB 这个项目。2023 年，我注意到他们的 Community Program 是有认真写的，不是无脑复制所谓成功项目的大段规则，于是开始跟相关成员探讨开源治理和社群运营的话题。后来，我读过 GreptimeDB 的源代码，发

https://www.tisonkun.org/2024/02/08/greptimedb-community-report/

**Comment by Vant1032** [comment]
作者写的文章真好，要是能在多个平台发布就好了。
我一直觉得要想长期保存个人信息，就得做好冗余，于是发表文章尽量多平台发布，这样从架构加都来讲可靠性更高
